src/Console/ModelsCommand.php

```diff
@@ -131,7 +135,8 @@ public function __construct(Filesystem $files)
      */
     public function handle()
     {
-        $filename = $this->option('filename');
+        $this->filename = $this->laravel['config']->get('ide-helper.models_filename');
```

Comment:
You need to provide the default here; otherwise ppl upgrade to the latest version and this returns `null`